porting/debug-build/udev.rst

```diff
@@ -1,16 +1,24 @@
 Add udev rules
 ==============
```

Comment:
Thanks, I think it can still be clearer

Please add three subsections: on device, on host, send upstream, i.e.:

Add udev rules
In order to allow some tests yadayadayada ...

On the device
Log in ... 

On the host
Alternatively, ...

Send upstream
In order for others ....